src/plugins/data_table/data_table.js

```diff
@@ -100,6 +100,26 @@ var LocalDataTable = (function() {
       return this.dataTable.fnSettings().fnRecordsTotal();
     },
 
+    hasUniqueValues: function() {
+      var hasUniques = false;
+
+      if (this.totalsRow && this.totalsRow.model) {
+        var totals = this.totalsRow.model.attributes;
+        var columns = this.totalsRow.model.keys();
+
+        _.each(columns, function(column) {
+          if (columns.indexOf(column + ".unique") !== -1) {
+            if (!(totals[column] instanceof Array) && totals[column] != totals[column + '.unique']) {
+              hasUniques = true;
+              return false;
+            }
+          }
+        });
+      }
```

Comment:
Totally agree. It was something I contemplated doing at first, but not sure why I went against it.